src/validator/client/README.md

```diff
@@ -2,7 +2,7 @@
 
 # 📘️ openapi-typescript-fetch
 
-A typed fetch client for [openapi-typescript](https://github.com/drwpow/openapi-typescript) 
+A typed fetch client for [openapi-typescript](https://github.com/drwpow/openapi-typescript)
```

Comment:
This whole package is actually "vendor" from a separate project, so I had opted to _not_ lint or format it. Actually, it should have been added to the ignored patterns for prettier and eslint. Buuuut, it looks like all the issues are auto-fixable, so I'll allow it in this case 😉 . Good idea.